python/lsst/analysis/ap/apdbUtils.py

```diff
@@ -51,10 +67,16 @@ def connectToApdb(dbName, dbType='sqlite', schema=None):
     elif dbType == 'postgres':
         if schema is None:
             raise RuntimeError('Schema must be set for postgres APDB')
-        host = 'lsst-pg-devel1.ncsa.illinois.edu'
+        if user is None:
+            user = 'rubin'
+        if host is None:
+            host = 'usdf-prompt-processing-dev.slac.stanford.edu'
         connection = psycopg2.connect(dbname=dbName,
                                       host=host,
+                                      user=user,
                                       options=f'-c search_path={schema}')
```

Comment:
Please delete this line, it doesn't work, and the `cursor.execute` line does what this was intending to do.